### DISCUSSION TITLE
Equipo Hackaton

### DISCUSSION THREAD

**Discussion by JulianVidal01** [Launch-X-Latam/MisionFrontEnd#555] [Team Posts]
Buenas a todos, Estoy en busca de equipo para la hackaton si desean pueden escribir, mi correo es lx11084@innovaccion.mx

**Comment by maurpz** [comment]
hola aun te faltan integrantes para los equipos de la hackaton ? soy
mienbro tambien este es mi correo de la institucion  ***@***.***
<http://innovaccion.mx/>*

**Comment by luisjacobpy** [comment]
Hola, ¿aún tienen espacio?, ando buscando equipo Mi email es: lx04550@innovaccion.mx
